packages/gatsby/src/commands/develop-process.ts

```diff
@@ -210,57 +230,104 @@ module.exports = async (program: IProgram): Promise<void> => {
       },
       doingEverythingElse: {
```

Comment:
Can we rename this state? It seems like it's not accurate anymore.

Reply:
Removed in #25716